### PR TITLE
Add clusterOSImage parameter to install config table

### LINKED ISSUE #21308: Explain install-config OSP clusterOSImage property
### Which section(s) is the issue in?

* https://docs.openshift.com/container-platform/4.3/installing/installing_openstack/installing-openstack-installer-custom.html#installation-configuration-parameters_installing-openstack-installer-custom

### What needs fixing?

Docs should note OS image override info: https://github.com/openshift/installer/blob/release-4.4/docs/user/openstack/customization.md#image-overrides

### PR DESCRIPTION
**Pull request by maxwelldb**
Resolves #21308

### CODE REVIEW

**Comment by maxwelldb** [comment]
For your review @Fedosin

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -312,6 +326,16 @@ ifdef::osp[]
    }
 }
 ----
+
+|`platform.openstack.clusterOSImage`
+|The location from which the installer downloads the {op-system} image.
+
+You may need to set this parameter to perform an installation in a restricted network.
```
Comment (racedo) [inline modules/installation-configuration-parameters.adoc:333]: It saus "you may need it" but this is required. I'd suggest something like the following:

If not specified, the installer will download the image from a predetermined location described in https://github.com/openshift/installer/blob/master/data/data/rhcos.json 

This is required to perform an installation in a restricted networking environment.

@Fedosin FYI

Comment (maxwelldb) [reply modules/installation-configuration-parameters.adoc:333]: Is there a reason to specify where the installer fetches the image from by default? If not, I'd prefer to leave it out--space is at a premium in the param tables. 

Will strike "may."

**Comment by maxwelldb** [comment]
Putting this on ice pending input from CS.

**Comment by maxwelldb** [comment]
FYI @vikram-redhat This is not intended for 4.4 GA.

**Comment by maxwelldb** [comment]
I'm incorporating this change into the in-progress restricted installation docs, e.g. #22478 

Closing.

**Pull request closed without merging**